sdk/storage/storage-file/execute-samples.js

```diff
@@ -0,0 +1,80 @@
+const execa = require("execa");
```

Comment:
Had thoughts of not duplicating the script, and keeping a common script for all the three packages but finally ended up duplicating anyway.
We need this `execa` library to run the scripts which means that it has to be present in the package.json as a devDependency.
Another option to avoid duplication is that we can tweak the script and keep it in the root of this repo so that all the packages can use that global script and pass in the `".env" file`, `"folder path"`, and the `language js/ts` options.